platform/security/sts/security-sts-certificateclaimshandler/src/main/java/org/codice/ddf/security/claims/certificate/CertificateClaimsHandler.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.claims.certificate;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.Principal;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.cxf.common.util.StringUtils;
+import org.apache.cxf.rt.security.claims.Claim;
+import org.apache.cxf.rt.security.claims.ClaimCollection;
+import org.apache.cxf.sts.claims.ClaimsHandler;
+import org.apache.cxf.sts.claims.ClaimsParameters;
+import org.apache.cxf.sts.claims.ProcessedClaim;
+import org.apache.cxf.sts.claims.ProcessedClaimCollection;
+import org.apache.cxf.sts.token.realm.RealmSupport;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CertificateClaimsHandler implements ClaimsHandler, RealmSupport {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CertificateClaimsHandler.class);
+
+    static final String EMAILADDRESS =
```

Comment:
We really should centralize these claim constants somewhere instead of duplicating them all over the place.